docs/contributing-documentation.md

```diff
@@ -1,54 +1,99 @@
-# Contributing documentation
+# Contributing to the HTML spec
```

Comment:
Changing the name breaks the naming convention used for all documentation files.

Reply:
Hmm. The problem is that "contributing documenation" seems to mean something different.

I'll look at the rest of the naming convention. Maybe that problem is more widespread...